srtcore/core.h

```diff
@@ -1053,6 +1053,30 @@ class CUDT
     std::pair<bool, time_point> packData(CPacket& packet);
 
     int processData(CUnit* unit);
+
+    /// This function passes the incoming packet to the initial processing
+    /// (like packet filter) and is about to store it effectively to the
+    /// receiver buffer and do some postprocessing (decryption) if necessary
+    /// and report the status thereof.
+    ///
+    /// @param incoming [in] The packet coming from the network medium
+    /// @param w_new_inserted [out] Set false, if the packet already exists, otherwise true (packet added)
+    /// @param w_was_sent_in_order [out] Set false, if the packet was belated, but had no R flag set.
+    /// @param w_srt_loss_seqs [out] Gets inserted a loss, if this function has detected it.
+    ///
+    /// @return 0 The call was successful (regardless if the packet was accepted or not).
+    /// @return -1 The call has failed: no space left in the buffer.
+    /// @return -2 The incoming packet exceeds the expected sequence by more than a length of the buffer (irrepairable discrepancy).
+    int handleSocketPacketReception(const std::vector<CUnit*>& incoming, bool& w_new_inserted, bool& w_was_sent_in_order, CUDT::loss_seqs_t& w_srt_loss_seqs);
+
+    // This function is to return the packet's play time (time when
+    // it is submitted to the reading application) of the given packet.
+    // This grp passed here by void* because in the current imp it's
+    // unused and shall not be used in case when ENABLE_BONDING=0.
+    time_point getPacketPlayTime(void* grp, const CPacket& packet);
```

Comment:
Still don't like this name. There is no "play time" in SRT. Why not just 1getPktTsbPdTime(..)`?

Requires `m_RcvBufferLock` to be locked.
```suggestion
    SRT_ATTR_EXCLUDES(m_RcvBufferLock)
    time_point getPacketPlayTime(void* grp, const CPacket& packet);
```